src/com/carolinarollergirls/scoreboard/core/Skater.java

```diff
@@ -35,10 +33,9 @@
     public void setPenaltyBox(boolean box);
     public String getFlags();
     public void setFlags(String flags);
-    public List<Penalty> getPenalties();
-    public Penalty getFOEXPPenalty();
     // A null code removes the penalty.
     public void penalty(String id, String number, int period, int jam, String code);
+    public void sortPenalties();
```

Comment:
This doesn't feel right, I'd expect this to be managed internal to the penalty handling.